docs/podman-run.1.md

```diff
@@ -21,6 +21,14 @@ If the IMAGE is not already loaded then **podman run** will pull the IMAGE, and
 all image dependencies, from the repository in the same way running **podman
 pull** IMAGE, before it starts the container from that image.
 
+Several files will be automatically created within the container when it is run.
```

Comment:
Aren't these files created at create/init time?  Perhaps "... within the container and then updated based on run time parameters when it is run."

Although to be honest I'm struggling with the wording here.

Reply:
Init does happen within `podman run`, so technically this is correct if we are talking about the run command? I don't know if there's an easy way to describe this bit within the manpage. Maybe we need a separate podman-containers manpage to describe things like this, so we can detail where in the lifecycle things happen?

Reply:
A lifecycle thing would be a great tutorial/blog post.  Just so folks don't pick nits, what would you think about just dropping "when it is run" from the first sentence.